[torch.export] Refactor placeholder_naming_pass to reduce CCN

malaybag opened the pull request:
Summary: Reduced CCN from 37 to 28 of placeholder_naming_pass method

Test Plan: Existing tests

Differential Revision: D85820388




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @Lucaskabela

angelayi reviewed (commented):
please land after landing the other PRs

malaybag committed:
[torch.export] Refactor placeholder_naming_pass to reduce CCN (#166600)

Summary:

Reduced CCN from 37 to 28 of placeholder_naming_pass method

Test Plan: Existing tests

Reviewed By: angelayi

Differential Revision: D85820388
